index.html

```diff
@@ -29,5 +29,116 @@ <h1>I'm Enssah. <br> Glad to see <br> you!</h1>
+                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                </ul>
+            </div>
+            <button class="see-mobile-btn" type="submit">See Projects</button>
+        </section>
+        <!-- card4 -->
+        <section class="card4">
+            <div class="mobileimage4"></div>
+            <h3>Multi-Post <br> Stories</h3>
+            <ul class="mobile">
+                <li class="company">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2013</li>
+            </ul>
+            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="teck">
+                <ul class="teck-list">
+                    <li><button class="mobile-btn" type="submit">HTML</button></li>
+                    <li><button class="mobile-btn" type="submit">CSS</button></li>
+                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                </ul>
+            </div>
+            <button class="see-mobile-btn" type="submit">See Projects</button>
+        </section>
+    </section>
+       <!-- about area -->
+        <section class="about">
+            <h5>About Myself</h5>
+            <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <span>LET'S CONNECT</span>
+            <ul class="media-links">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li> 
+            </ul>
+            <button class="resume-mobile-btn" type="submit">Get my resume</button>
+            <div class="lang-area">
+                <div class="stacknav">
+                    <strong>Languages</strong>
+                    <i class="fa-solid fa-angle-down"></i>
+                </div>
+                <div class="stacks">
+                    <button class="html" type="submit"><a href="#">HTML</a></button>
+                    <button class="css" type="submit"><a href="#">CSS</a></button>
+                    <button class="js" type="submit"><a href="#">JavaScript</a></button>
+                </div>
+                <div class="frameworks">
+                    <strong>Frameworks</strong>
+                    <i class="fa-solid fa-angle-right"></i>
+                </div>
+                <div class="skills">
+                    <strong>Skills</strong>
+                    <i class="fa-solid fa-angle-right"></i>
+                </div>
+            </div>
```

Comment:
- [ ] For semantics purpose, kindly wrap the technology categories `(Languages, Frameworks and Skills) ` in a `ul`, each of them in a `li` and the tech stacks in each category `( JavasScript, HTML and CSS)` are also a list ul and each of them a` li.` This will creates a visual indentation and helps organize and present the information in a structured manner. ℹ️ 
```suggestion
<ul>
    <li>Languages
      <ul>
      <li>Sub-item 2.1</li>
      <li>Sub-item 2.2</li>
      <li>Sub-item 2.3</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>

```